src/runtime_src/core/edge/user/aie/graph.cpp

```diff
@@ -421,6 +504,13 @@ xrtGraphUpdateRTP(xrtGraphHandle ghdl, const char* port, const char* buffer, siz
   graph->update_rtp(port, buffer, size);
 }
 
+void
+xrtSyncBOAIE(xrtGraphHandle ghdl, unsigned bo, const char *dmaID, enum xclBOSyncDirection dir, size_t size, size_t offset)
```

Comment:
This is just my pet peeve, I have never liked bare 'unsigned'.   Please use 'unsigned int' to be explicit.